lib/link.sh

```
@@ -1,30 +1,31 @@
+source lib/ui.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)